app/views/user/settings.ctp

```diff
@@ -36,282 +36,318 @@ $this->set('title_for_layout', $pages->formatTitle(__('Settings', true)));
 </div>
 
 <div id="main_content">
-    <div class="module options">
+    <div class="options">
         <?php echo $form->create(null, array('action' => 'save_settings')); ?>
+        <div md-whiteframe="1" class="settings-form">
+            <h2><?php __('Options'); ?></h2>
+            <fieldset>
+            <md-list flex role="list" class="flex" >
+                <md-list-item>
+                    <div>
```

Comment:
This is what happens when I remove the div.

![image](https://cloud.githubusercontent.com/assets/10293591/16387080/ebdd899e-3c58-11e6-8ecf-eb9f2bddc023.png)